algo/README.md

```diff
@@ -449,8 +450,10 @@ elsewhere.
 
 ## Version History and Download Links
 
+- Upcoming:
+    - Add "Decreased response" and "No benefit" to directions to output
```

Comment:
This is not an entirely correct sentence I think! Maybe just say "to evidence directions" and remove "to output"?